### PR TITLE
{chem}[foss/2020b,intel/2020b] PyRETIS v2.5.0 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires #12282 (MDTraj), ~~#12283~~ (PyTables)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3521.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/5cee26b502974edb0f0ed5f407f4d058 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3152.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/1e932f6b6ae963f8f5c79eb2abedd631 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node2685.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/60260c3177c96dd4597a5fb819266f6e for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/ad1451054f6adeab27b12b2502474ff2 for a full test report.

File: easybuild/easyconfigs/p/PyRETIS/PyRETIS-2.5.0-intel-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+sanity_check_paths = {
+    'files': ['bin/pyretisanalyse', 'bin/pyretisrun'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = [
+    "pyretisanalyse --help",
+    "pyretisrun --help",
+]
+
```
Comment (Micket) [inline easybuild/easyconfigs/p/PyRETIS/PyRETIS-2.5.0-intel-2020b.eb:43]:
```suggestion

sanity_pip_check = True

```

File: easybuild/easyconfigs/p/PyRETIS/PyRETIS-2.5.0-foss-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+sanity_check_paths = {
+    'files': ['bin/pyretisanalyse', 'bin/pyretisrun'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = [
+    "pyretisanalyse --help",
+    "pyretisrun --help",
+]
+
```
Comment (Micket) [inline easybuild/easyconfigs/p/PyRETIS/PyRETIS-2.5.0-foss-2020b.eb:43]:
```suggestion

sanity_pip_check = True

```

File: easybuild/easyconfigs/p/PyRETIS/PyRETIS-2.5.0-intel-2020b.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/p/PyRETIS/PyRETIS-2.5.0-intel-2020b.eb:43]: I'm just gonna apply these since I don't think there is any need to discuss them.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 7 out of 7 (2 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/2f58e4e88ea838e7da3c0dc8d4c6a540 for a full test report.

**Pull request merged**